charts/grafana/FLUX.md

```diff
@@ -1,12 +1,32 @@
 # Flux language setup
-### Manual Setup
-* Flux is still in beta, so this information may be out of date
-* Follow instructions laid out by [Flux Datasource for Grafana]
-(https://grafana.com/plugins/grafana-influxdb-flux-datasource)
+## Manual Setup
+- [Flux](https://docs.influxdata.com/flux/v0.12/) is still in beta, so this information may be out of date
+- [Enable](https://docs.influxdata.com/flux/v0.12/introduction/installation) Flux by setting the `flux-enabled` option to `true` under the `http` section of your influxdb.conf
+- Follow instructions laid out by [Flux Datasource for Grafana](https://grafana.com/plugins/grafana-influxdb-flux-datasource)
 
+## Cronus-Monitoring Setup
+- Run the init.sh script in the scripts directory and the included helm chart will enable flux automatically
+    > Flux is enabled in Grafana via the [values.yaml](https://github.com/Eliforbes42/cronus-monitoring/blob/master/charts/grafana/values.yaml) `datasources` with `grafana-influxdb-flux-datasource` as the type
+    > Flux is enabled in InfluxDB via the [values.yaml](https://github.com/Eliforbes42/cronus-monitoring/blob/master/charts/influxdb/values.yaml) `http` with `flux_enabled` being set to true and [influxdb/templates/config.yaml](https://github.com/Eliforbes42/cronus-monitoring/blob/master/charts/influxdb/templates/config.yaml) taking that value and deploying the flux enabled InfluxDB
 
-### Helpful information
-* Read more about Flux [here](https://github.com/influxdata/platform/tree/master/query)
-* Use the limit function to reduce the number of datapoints so that the master node of the kubernetes cluster doesn't get dropped
+## Helpful information
+- Read more about [Flux](https://github.com/influxdata/platform/tree/master/query)
 
+## How to begin writing queries in grafana
+1. Open the dashboard you wish to work with
```

Comment:
Repetition is no good

Reply:
I  updated this by making the first line about going through dashboard->panel creation->panel editor